spec/ruby_memoized/memoizer_spec.rb

```diff
@@ -15,41 +15,46 @@
 
   let(:method) { :american_exceptionalism }
 
-  subject { memoizer.call(*args, &block) }
+  subject { memoizer.call(*args, **kwargs, &block) }
```

Comment:
On line 12 above (in this file), do you have to do some similar double splatting on the class definition you've added? https://github.com/kklimuk/ruby_memoized/pull/2/files#diff-83353dc8cb2fcf868660eccb407f798fR12 (just comparing against what has been changed, but not sure).

Reply:
Got it. I guess what I was saying was do you need to change `define_method(method_name) { |*_, &_| output_value }` to `define_method(method_name) { |*_, **_, &_| output_value }` or something like that. 

Not sure, but just another place where it feels like there's a reference to the old style `*args, &block` that has now been changed to `*args, **kwargs, &block` everywhere (obviously with the caveat that it's underscore non-used methods).

Reply:
ah, i see your point

Reply:
fixed